docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md

```diff
@@ -3,19 +3,33 @@ rss_title: Codacy release notes RSS feed
+As part of an ongoing effort to improve the speed and value of the insights provided by Codacy, we've been working on a new Coverage engine and started its deployment on November 23rd, 2023. The transition to use the new engine across Codacy will take several months and will eventually impact all the coverage data you see on the UI, API, and Git provider.
 
-On November 23rd 2023 we activated the new, faster Coverage engine and set it to send coverage data to your Git provider. This is part of an ongoing effort to improve the speed and value of the insights provided by Codacy.
+As a result, both old and new data will coexist during this period, <span class="skip-vale">potentially</span> leading to [differences in reported metrics](#differences-in-coverage-metrics-between-the-old-and-new-coverage-engines). Please refer to the table below for the updated status of the transition process. The table will be updated as changes are introduced.
 
-As a consequence of this update, you can now see two additional checks on your pull requests, marked **beta**, alongside the preexisting **Codacy Coverage Report** check. If you are using the Codacy Coverage Report status check to block merging pull requests on GitHub, please [update your setup](#if-you-are-using-the-old-coverage-status-check-to-block-merging-pull-requests-on-github) and avoid disruptions in the future.<!--NOTE Temporary comment for review purpose, to remove-->
+| Codacy feature             | Transition status | Notes                                                                                                                                        |
+|----------------------------|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| Codacy app UI              | Planned           | -                                                                                                                                            |
+| Codacy API                 | Planned           | -                                                                                                                                            |
+| Git provider status checks | Beta              | Data from the new Coverage engine is marked \[beta\]<br>and shown alongside data from the current engine. [Read more below](#status-checks). |
+| GitHub coverage summaries  | Planned           | -                                                                                                                                            |
```

Comment:
@codacy/alabama As mentioned during the daily, I'm updating this page to give more visibility to the user what's happening with the transition to the new engine. I'll then update this table as needed as the updates go online.

Let me know if I should add more entries here, if this approach is overkill, or any other observations.

cc @machadoit

Reply:
I noticed that we didn't have a doc task on the summary changes https://codacy.atlassian.net/browse/ALA-589 we are now putting it live as time of writing. 

So we can update the docs to 'Live' on the check? With whatever notes you see suitable. Or if you prefer, feel free to add the task on the epic to update on a different PR.

Overall the changes look pretty cool! 👌